dashboard/app/views/layouts/_header.html.haml

```diff
@@ -65,7 +65,9 @@
   - view_as = @user || current_user
   - user_progress = current_user == nil ? 'null' : summarize_user_progress(script_level.script, view_as, view_as == current_user && @public_caching).to_json
 
+  - page = params[:page] || -1
```

Comment:
hm, I just realized that it may be a bad idea to use page as the name for this param because this is partial is included in almost every page on the site, including others which might use page for pagination (like the gallery). Would suggest changing it to something more specific to avoid current and future potential conflicts (and maybe make it easier for a future reader to figure out what this "page" thing is).